moduleframework/environment_prepare/docker_prepare.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Jan Scotka <jscotka@redhat.com>
+#
+
+"""
+module for environment setup and cleanup, to be able to split action for ansible, more steps instead of one complex
+"""
+
+from avocado.utils import service
+from moduleframework.common import *
+
+class EnvDocker(CommonFunctions):
+
+    def prepare_env(self):
+        self.loadconfig()
+        self.__install_env()
+        self.__start_service()
+
+    def cleanup_env(self):
+        self.__stop_service()
+
+    def add_insecure_registry(self, registry):
+        """
+        Append registry into inserure registry.
```

Comment:
actually this function is not used anyhow, because of moving this setup out of testing. Could you write issue, to do that. we can solve it in separate issue.